django-markdown/django_markdown/django_markdown_app/markdown_extensions.py

```diff
@@ -0,0 +1,20 @@
+import markdown
+from markdown.inlinepatterns import LinkInlineProcessor, LINK_RE
+from django.urls import reverse
```

Comment:
Probably need to run [isort](https://pypi.org/project/isort/) on your codebase to ensure consistent import statement ordering, which complies with PEP 8.